README.md

```diff
@@ -218,6 +221,7 @@ To check the status of a submitted job, just use the suggested command:
 
 ```bash
 cloudos job status \
+    -c $CLOUDOS \
     --apikey $MY_API_KEY \
     --cloudos-url https://cloudos.lifebit.ai \
```

Comment:
```Suggestion
```

we have the flag `-c $CLOUDOS`, this now can be removed.
let's use the explicit flag for everything in the README.md